playbooks/roles/oauth_client_setup/tasks/main.yml

```diff
@@ -33,7 +33,7 @@
     --client_secret {{ item.secret }}
     --trusted
     --logout_uri {{ item.logout_uri | default("") }}
-    --username {{ item.username }}
+    {% if item.username is defined %} --username {{ item.username }} {% endif %}
```

Comment:
Is the `is defined` super needed? can't we just use `{% if item.username %}`

Reply:
No, I tried that first, and I was getting an error, "The username key doesn't exists" or something like that. We need to explicitly check if it's defined.